### DISCUSSION TITLE
Sending hex commands?

### DISCUSSION THREAD

**Discussion by Ansuo** [roleoroleo/yi-hack-Allwinner-v2#201] [Q&A] [answered]
I want to send hex commands to the cam. It's to test some app options. I guess ipc_cmd is for that case?

For testing and better understanding I want to send `ipc_cmd -r on` as hex command:
`02 00 00 00 08 00 00 00 7a 00 01 00 00 00 00 00 `
`"\x02\x00\x00\x00\x08\x00\x00\x00\x7a\x00\x01\x00\x00\x00\x00\x00"`

Is `ipc_cmd -x` the correct argument? Which of the above hex values should be used?

**Comment by roleoroleo** [comment, marked as answer]
Use -f option with a binary file.

**Comment by Ansuo** [comment]
thx mate 👍
